src/request_parser.ts

```diff
@@ -62,9 +63,10 @@ export const parseTakerRequest = (req: Pick<express.Request, 'headers' | 'query'
             } else {
                 takerRequest = {
                     ...takerRequestBase,
-                    sellAmountBaseUnits: new BigNumber(query.sellAmountBaseUnits),
                     protocolVersion,
+                    sellAmountBaseUnits: new BigNumber(query.sellAmountBaseUnits),
                     txOrigin: query.txOrigin!,
+                    isLastLook,
```

Comment:
Why not add this in `takerRequestBase`

Reply:
As I understand it `isLastLook` is only for v4 - so I only added it to the v4 version. But can add to `takerRequestBase` if needed for v3 and v4

Reply:
oh I see it! thanks that makes sense.

Would you be so kind to create a mixin object that we can use to merge v4-related taker parameters without having to duplicate them?

Reply:
Done!